wgpu-core/src/command/render.rs

```diff
@@ -259,10 +259,7 @@ impl VertexState {
         self.vertex_limit = !0;
         self.instance_limit = !0;
         for vbs in &self.inputs {
-            if vbs.stride == 0 {
-                continue;
-            }
-            let limit = (vbs.total_size / vbs.stride) as u32;
+            let limit = vbs.total_size as u32;
```

Comment:
why is this change done? the limit is supposed to be on the number of vertices. If we ignore the stride, we are counting the bytes now.

Reply:
I also have some doubts about this but some time total_size contains the number of vertices instead of bytes

Reply:
there is only one place where it's assigned, and it's in bytes:
```rust
state.vertex.inputs[slot as usize].total_size = match size {
                        Some(s) => s.get(),
                        None => buffer.size - offset,
                    };
```

Reply:
You're right I was confusing things